src/client/sandbox/event/focus-blur.js

```diff
@@ -150,10 +160,24 @@ export default class FocusBlurSandbox extends SandboxBase {
         if (el[type]) {
             // NOTE: We should guarantee that activeElement will be changed, therefore we need to call the native
             // focus/blur event. To guarantee that all focus/blur events are raised, we need to raise them manually.
-            var windowScroll = null;
+            var windowScroll                     = null;
```

Comment:
This method is already enormously huge. Can we refactor this chunk of code into separate method?